src/test/scala/org/graphframes/ldbc/LDBCSuiteBFS.scala

```diff
@@ -0,0 +1,55 @@
+package org.graphframes.ldbc
+
+import org.graphframes.{GraphFrame, GraphFrameTestSparkContext, SparkFunSuite}
+import org.graphframes.examples.LDBCUtils
+
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.types.LongType
+
+class LDBCSuiteBFS extends SparkFunSuite with GraphFrameTestSparkContext {
+  test("Test LDBC BFS") {
+    assume(sys.env.contains("LDBC_TEST_ROOT"))
+    LDBCUtils.downloadLDBCIfNotExists()
+
+    val enabled = spark.conf.getOption("spark.sql.adaptive.enabled")
+    try {
+      // disable AQE
+      spark.conf.set("spark.sql.adaptive.enabled", value = false)
```

Comment:
Why disable AQE? I’ve found some algorithms can be painfully slow without it. Avoid bugs?

Reply:
This PR at the moment is for testing / sharing / discussions. I was trying to disable the AQE after thread in mailing list